spec/virtual_attributes_spec.rb

```diff
@@ -983,4 +983,13 @@ def col2
     tc.save
     expect(tc.attributes.size).to eq(2)
   end
+
+  it "doesn't botch up the attributes when requiring", :with_test_class do
```

Comment:
referencing?  I'm not sure what you mean by requiring

Reply:
stupid `s/require/references/g`